libs/core/src/lib/dynamic-page/dynamic-page.component.ts

```diff
@@ -317,4 +310,27 @@ export class DynamicPageComponent implements AfterViewInit, OnDestroy, WithDynam
             );
         }
     }
+
+    /** @hidden */
+    private _setContentFooterSpacer(): void {
+        this._contentComponent.changes
+            .pipe(
+                startWith(this._contentComponent.toArray()),
+                // to avoid NG0100: Expression has changed after it was checked
+                debounceTime(0),
```

Comment:
just FYI, it makes more sense to use `observeOn(asyncScheduler)` as we don't need to filter anything (which is the main purpose of debounceTime)